pytrip/cube.py

```diff
@@ -918,3 +918,13 @@ def write_trip_data(self, path):
             cube = cube.byteswap()
         cube.tofile(path)
         return
+
+    @classmethod
+    def save(cls, path):
+        """ Saves the object as a pickle object.
+        :params: path where file is stored.
+        """
+
+        import pickle
+        with open(path, 'wb') as f:
+            pickle.dump(cls, f, pickle.HIGHEST_PROTOCOL)
```

Comment:
Is it compatible between python 2 and 3 ?
If you save in python3 and load in python2, will it work ?

Python 3.4 seems to introduce protocol versions which are not available in Python2:
https://docs.python.org/3/library/pickle.html#data-stream-format